File: GildedRose/ruby/gilded_rose_spec.rb
    
require File.join(File.dirname(__FILE__), 'gilded_rose')


describe GildedRose do

 describe "#update_quality" do
   items = [Item.new("foo", 0, 0)]
   GildedRose.new().update_quality(items)
   items[0].name.should == "fixme"
 end

end

File: haskell/test/GildedRoseSpec.hs
module GildedRoseSpec (spec) where

import Test.Hspec
import GildedRose

spec :: Spec
spec =
  describe "updateQuality" $ do

    it "fixme" $
       let inventory = [Item "foo" 0 0]
           actual = updateQuality inventory
           expected = []
       in actual `shouldBe` expected
